content/internships/contents.lr

```diff
@@ -11,13 +11,15 @@ body:
 ### Programs
 
 **Google Summer of Code**  
-CC has applied to be a mentoring organization for [Google Summer of Code](https://summerofcode.withgoogle.com/) 2020. We will hear back on 2020-02-20 whether we've been accepted.
-
-Feel free to take a look at our [Project Ideas](/internships/project-ideas) for the program. Please note that these project ideas are shared between all current internship programs.
+CC is participating in [Google Summer of Code](https://summerofcode.withgoogle.com/) 2020. [Here is our organization page on the GSoC website](https://summerofcode.withgoogle.com/organizations/5742311717208064/).
 
 **Outreachy**  
 Creative Commons is participating in the [Outreachy](https://www.outreachy.org/) May-August 2020 round. [Here is our community page on the Outreachy website](https://www.outreachy.org/communities/cfp/creative-commons/).
- 
+
+----
+
+Feel free to take a look at our [Project Ideas](/internships/project-ideas). Please note that these project ideas are shared between all current internship programs. 
+
```

Comment:
_Project Ideas_ is not a program, so I don't think it makes sense as a fake header there. I moved it to its own real header.

I'm not worrying about the 79 character limit because it would be inconsistent with the rest of the codebase and this is not Python.